lib/suspenders/app_builder.rb

```diff
@@ -467,6 +424,10 @@ def configure_rubocop
       template '.rubocop.yml', '.rubocop.yml'
     end
 
+    def run_stairs
+      bundle_command 'exec stairs --use-defaults'
```

Comment:
@patbenatar does stairs run `bundle install`? if not should we throw that in here before running stairs?